### ISSUE TITLE
Configuring to work with google colab

### ISSUE THREAD

**Issue by SuperGayBoi** [lukalabs/cakechat#48]
how to add support to train using Google Colab to take advantage of cloud computing power and train better models?

**Comment by nicolas-ivanov** [comment]
@SuperGayBoi please, see this tutorial for a reference: https://www.kdnuggets.com/2018/02/google-colab-free-gpu-tutorial-tensorflow-keras-pytorch.html/2
